lib/players/computer.rb

```diff
@@ -0,0 +1,50 @@
+require 'pry'
+
+module Players
+
+  class Computer < Player
+
+    WIN_COMBINATIONS = [
+      [0,1,2], # top row
+      [3,4,5], # middle row
+      [6,7,8], # bottom row
+      [0,3,6], # left column
+      [1,4,7], # middle column
+      [2,5,8], # right column
+      [0,4,8], # left diagonal
+      [6,4,2] # right diagonal
+    ]
+
+    def move(board)
+      # Computer will first look to see if there any of the winning combinations can be completed on the next move.
+      # Didn't attempt to have computer choose to win vs. block. Computer will complete the first winning combination, regardless of token.
+
+      WIN_COMBINATIONS.detect do |combo|
+        # Why doesn't this work without "return"?
```

Comment:
Prefer to keep game-related logic in the Game class

Reply:
I was trying to figure out the best way to access WIN_COMBINATIONS without adding it in the Computer class. Any suggestions on how to do that?